articles/client-platforms/react/06-rules.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Cannot see this page in the deployed app: https://auth0-docs-content-pr-1763.herokuapp.com/docs/quickstart/spa/react

You probably need to update the `index.yml` with the new page: https://github.com/auth0/docs/blob/master/articles/client-platforms/react/index.yml